initialscreen.php

```diff
@@ -0,0 +1,165 @@
+<?php
```

Comment:
This page does not work.  It does not load in the browser.  Please fix and resubmit for further review.  No further comments until the page is fixed.